login-workflow/src/screens/EulaScreen/EulaScreenBase.tsx

```diff
@@ -51,12 +53,24 @@ export const EulaScreenBase: React.FC<EulaScreenProps> = (props) => {
         <WorkflowCard {...cardBaseProps}>
             <WorkflowCardHeader {...headerProps} />
             <WorkflowCardBody>
-                {!htmlEula && <Box sx={{ flex: '1 1 0px', overflow: 'auto' }}>{eulaContent}</Box>}
+                {/* {!htmlEula && <Box sx={{ flex: '1 1 0px', overflow: 'auto' }}>{eulaContent}</Box>}
                 {htmlEula && (
                     <Box
                         sx={{ flex: '1 1 0px', overflow: 'auto' }}
                         dangerouslySetInnerHTML={{ __html: DOMPurify.sanitize(eulaContent as string) }}
                     />
+                )} */}
+                {checkboxProps.disabled ? (
+                    <Button variant="outlined" color="primary" onClick={onRefetch} sx={{ mt: 'auto', mb: '50%' }}>
+                        Refetch
+                    </Button>
```

Comment:
Let's use a centered `<Autorenew />` icon button instead of this full width "refetch" text button. @huayunh will that suffice or do you want to provide some better UI/UX around this?

Reply:
Agree with Tom. Considering having something like this centered in the div instead:

![image](https://github.com/etn-ccis/blui-react-workflows/assets/8997218/4c54f20b-b52b-4fcb-ab1a-646aebdeb9b3)

Reply:
(Icon size 36dp, font - subtitle2)